CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
-## 0.14.4-dev2
+## 0.14.4-dev
```

Comment:
dev3? :)

Reply:
ohh right, we increment these over here!